client/reader/stream/x-post.jsx

```diff
@@ -80,9 +80,10 @@ class CrossPost extends PureComponent {
 
 	handleClick = ( event ) => {
 		if ( event.button > 0 || event.metaKey || event.controlKey || event.shiftKey || event.altKey ) {
-			return;
+			window.open( this.props.post.URL, '_blank', 'noopener noreferrer' );
```

Comment:
For example, imagine we make a separate component called `Byline` (true story).  It has its own onClick that competes with the cards onClick. 

Now its really really easy to introduce a bug where when you ctrl click on the card's text area (not title) we accidentally preventDefault when we shouldn't have (bug that exists in prod)